### ISSUE TITLE
Changed branch instruction still shows a jump arrow

### ISSUE THREAD

**Issue by ViRb3** [x64dbg/x64dbg#1219]
Not a big deal and should be simple to fix. There was a JE instruction before, where the arrow starts.

![](https://i.imgur.com/gT6tt7C.png)

**Comment by mrexodia** [comment]
yes, xref analysis will be snown regardless if whether the instruction is still present. You can re-analyze the currently selected function with the command `analr func.start(dis.sel())` you can bind it to a hotkey with the favorite tools if you want.

**Comment by ViRb3** [comment]
**Ctrl + A** to re-analyze the whole module also works, although it could take a long time on a big module.

**Comment by mrexodia** [comment]
Yeah, a cache is used to show xrefs. Analyzing the function/module or even executing the `xanal` command will invalidate the cache and show the new lines. The only reason this is not closed is that the GUI can check if the lines are correct before actually showing them.
